.travis.yml

```diff
@@ -29,6 +29,8 @@ matrix:
     env: KAFKA_VERSION=2.6.0
   - scala: 2.13
     env: KAFKA_VERSION=2.7.1
+  - scala: 2.13
+    env: KAFKA_VERSION=2.8.1
```

Comment:
can you please also update LATEST: https://github.com/wurstmeister/kafka-docker/blob/0396ad2b104146b56244d1cc273ad1e1fc975490/.travis.yml#L11